src/main/java/com/peerrequest/app/api/DirectRequestsController.java

```diff
@@ -0,0 +1,264 @@
+package com.peerrequest.app.api;
+
+import com.peerrequest.app.data.DirectRequest;
+import com.peerrequest.app.data.DirectRequestProcess;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+
+/**
+ * The RequestsController handles all directRequests interaction with the backend.
+ */
+@RestController
+@ApiControllerPrefix
+public class DirectRequestsController extends ServiceBasedController {
+
+    private final int maxPageSize = 100;
+
+    /**
+     * Gets a directRequestProcess.
+     *
+     * @param entryId entry of the directRequestProcess
+     *
+     * @return directRequestProcess of entry
+     */
+    @GetMapping(value = "/categories/{category_id}/entries/{entry_id}/process", produces = "application/json")
+    public DirectRequestProcess.Dto getDirectRequestProcess(@PathVariable("entry_id") final Long entryId) {
+        var option = this.directRequestProcessService.get(entryId);
+        if (option.isEmpty()) {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "direct request process does not exist");
+        }
+
+        return option.get().toDto();
+    }
+
+    /**
+     * Creates a directRequestProcess.
+     *
+     * @param entryId entry of the directRequestProcess
+     * @param dto containing the open slots
+     *
+     * @return the added directRequestProcess
+     */
+    @PostMapping(value = "/categories/{category_id}/entries/{entry_id}/process", produces = "application/json")
```

Comment:
remove:  produce = "aplication/json"
to match the other controllers

Reply:
Done.